tests/integration/test_utils/data_source_config/sql.py

```diff
@@ -113,6 +130,11 @@ def setup(self) -> None:
     def teardown(self) -> None:
         for table in self.tables:
             table.drop(self.engine)
+        with self.engine.connect() as conn, conn.begin():
```

Comment:
pytest question: Are we guaranteed teardown will get called, even if the test errors out? I want to make sure we tear down the schema.

Reply:
As long as the test errors out, rather than something in the fixtures. There are ways we can pollute things, but that's been the case for a while with other tests as well. I think we should be covered for most cases. I'll share out some more details in slack.